components/button/README.md

```diff
@@ -1,5 +1,20 @@
 # Buttons
 
+## d2l-button
+
+The `d2l-button` element can be used just like the native button element, but also supports the primary attribute for denoting the primary button.
+
+![Button](./screenshots/button.png?raw=true)
+
+### Usage
+
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/button/button.js';
+</script>
+<d2l-button text="My Button" icon="d2l-tier1:gear"></d2l-button>
```

Comment:
`d2l-button` uses a slot instead of text and icon like in subtle button, so this should be more like `<d2l-button>My Button</d2l-button>`